polyphemus/lib/polyphemus/controllers/job_controller.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'controller'
+require_relative '../../etls/redcap/redcap_etl_script_runner'
+
+require 'stringio'
+
+# Add this Kernel method to capture
+#   the stdout messages from the ETL
+#   process. We'll stream those back
+#   to the consumer, since loading
+#   can be a long process.
+# This will probably not be necessary
+#   once we migrate to an async job
+#   scheduler.
+module Kernel
+
+  def capture_stdout
+    out = StringIO.new
+    $stdout = out
+    yield
+    return out
+  ensure
+    $stdout = STDOUT
+  end
+
+end
+
+class JobController < Polyphemus::Controller
```

Comment:
Nice, I like this.  With this approach, it would mean we can put the run redcap command in etna instead of polyphemus due to the lessened gem requirements?